### PR TITLE
Fix loadReactorState

### PR DESCRIPTION
**Pull request by atchapcyp**
- [ ] I added unit tests for any code that added
- [ ] I updated the CHANGELOG.md 
- [ ] All IP is original and not copied from another source
- [ ] I assign all copyright to Loom Network for the code in the pull request

### CODE REVIEW

File: fnConsensus/store.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ func loadReactorState(db dbm.DB) (*ReactorState, error) {
 	if err := persistedRectorState.Unmarshal(rectorStateBytes); err != nil {
 		return nil, err
 	}
+	persistedRectorState.Messages = make(map[string]Message)
```
Comment (enlight) [inline fnConsensus/store.go]: This should go into `ReactorState.Unmarshal`

Comment (atchapcyp) [reply fnConsensus/store.go]: Got it.

File: fnConsensus/reactor.go [hunk 2]
```diff
@@ -342,6 +347,12 @@ OUTER_LOOP:
 }
 
 func (f *FnConsensusReactor) voteRoutine() {
+	defer func() {
+		if r := recover(); r != nil {
+			f.Logger.Error("Recovered in Reactor.voteRoutine ", r)
```
Comment (enlight) [inline fnConsensus/reactor.go]: `f.Logger.Error("Recovered in FnConsensusReactor.voteRoutine", "r", r)`

File: fnConsensus/reactor.go [hunk 3]
```diff
@@ -304,6 +304,11 @@ func (f *FnConsensusReactor) initRoutine() {
 }
 
 func (f *FnConsensusReactor) commitRoutine() {
+	defer func() {
+		if r := recover(); r != nil {
+			f.Logger.Error("Recovered in Reactor.commitRoutine")
```
Comment (enlight) [inline fnConsensus/reactor.go]: `f.Logger.Error("Recovered in FnConsensusReactor.commitRoutine", "r", r)`

File: fnConsensus/fnConsensus_test.go [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+	"github.com/stretchr/testify/require"
+	dbm "github.com/tendermint/tendermint/libs/db"
+)
+
+func TestUnmarshalReactorState(t *testing.T) {
+	db := dbm.NewMemDB()
+	rs := NewReactorState()
+
+	rsByte, err := rs.Marshal()
```
Comment (enlight) [inline fnConsensus/fnConsensus_test.go]: Should do `require.NoError(t, err)` here too

**Pull request merged**